app/imports/components/projects/AddProject.jsx

```diff
@@ -133,6 +185,16 @@ export class AddProject extends Component {
     this.setState(this.getInitalState());
   }
 
+  callAddProject(project) {
+    addProject.call(project, (err, res) => {
+      if (err) {
+        console.error(err.error);
+      } else {
+        this.cleanForm();
```

Comment:
It will re route to single project view but it is not ready, so i think it's ok like this for now.